app/assets/javascripts/application.js

```diff
@@ -12,5 +12,6 @@
 //
 //= require rails-ujs
 //= require jquery
+//= require jquery_ujs
```

Comment:
I thought that rails-ujs was a replacement for jquery-ujs. What do you notice changes? https://gorails.com/forum/do-i-need-rails-ujs-and-jquery_ujs

Reply:
Only that the [IE8 errors here](https://trello.com/c/5k9FhSCK/63-cross-browser-testing) are fixed when `jquery-ujs` is included…